src/terraform/index.go

```diff
@@ -10,6 +10,10 @@ import (
 	"github.com/pkg/errors"
 )
 
+// TerraformModulesCommitHash is the git commit hash that reflects
+// which of the Terraform modules in Originate/exosphere we are using
+const TerraformModulesCommitHash = "8786f912"
```

Comment:
I don't think this needs to be public since it appears to only appear in this package. Thus it could become: `terraformModulesCommitHash`